www/vorj00/ukol_03/assets/css/cv/style.css

```diff
@@ -0,0 +1,125 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather');
+
+body {
+    font-family: 'Merriweather', serif;
+    color: #FAFAFA;
+    font-size: 16px;
+}
+
+.main>* {
+    padding: 1.5rem;
+    width: calc(100% - 3rem);
+}
+
+.main>*>h3 {
```

Comment:
how about .main h3 ? will you have h3 any where else in .main?

avoid specificity, you are limiting urself

je tohle generovan v SCSS?jestli ano tak chci zdrojak SCSS a ne vystup

Reply:
je to čistý CSS, ale přišlo mi logické u tohohle ji zachovat — pokud by se jinde objevila h3, tak už nebude sticky a nemusela by se chovat podle očekávání